Public/Src/Engine/Scheduler/Fingerprints/PipFingerprinter.cs

```diff
@@ -50,9 +50,15 @@ public abstract class PipFingerprinter
         /// </summary>
         public delegate PipData PipDataLookup(FileArtifact artifact);
 
+        /// <summary>
+        /// Refers to a function which maps a process to its source change affected dependencies. 

```

Comment:
>source [](start = 61, length = 6)

its inputs -- sources are source files, but inputs can be produced by another process.

Reply:
Let me add - in between the words. source-change-affected